zivid_camera/src/settings_generator.cpp

```diff
@@ -49,6 +50,23 @@ struct DependentFalse : std::false_type
 
 void writeToFile(const std::string& file_name, const std::string& text)
 {
+  if (std::filesystem::exists(file_name))
+  {
+    std::ifstream file(file_name);
+    if (!file || !file.is_open())
```

Comment:
Nice to avoid the unnecessary re-build here :+1: 
Curious; Is it needed to have both checks in `!file || !file.is_open()`?

Reply:
no it is not needed to have both checks, either one will work, i can change to just is_open.